media-sound/milkytracker/milkytracker-1.02.00.ebuild

```diff
@@ -1,13 +1,18 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
 inherit cmake desktop
 
+# This commit is needed so the milkytrace binary is linked properly
+# Bug 711562
+COMMIT="2b145b074581ddf3b4ad78a402cdf5fab500b125"
+
 DESCRIPTION="FastTracker 2 inspired music tracker"
 HOMEPAGE="https://milkytracker.titandemo.org/"
-SRC_URI="https://github.com/milkytracker/MilkyTracker/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/milkytracker/MilkyTracker/archive/v${PV}.tar.gz -> ${P}.tar.gz
```

Comment:
I guess the cmake patch is too big to include here?

Reply:
Yes, ~40kb, `repoman` complained about it.